pkgs/top-level/all-packages.nix

```diff
@@ -37887,7 +37887,7 @@ with pkgs;
 
   wmutils-opt = callPackage ../tools/X11/wmutils-opt { };
 
-  wordpress = callPackage ../servers/web-apps/wordpress { };
+  inherit (callPackage ../servers/web-apps/wordpress {}) wordpress wordpress6_1;
```

Comment:
I don't think we need wordpress here. We can just update the package option of the module.